docs/README.md

```diff
@@ -1044,6 +1044,133 @@ docker-compose up
 2. Site will be avaiable on `localhost:3001`
 3. You can update the documentation by modifying the file `docs/README.md`
 
+## OpReturn
+
+OP_RETURN is a [script](https://wiki.bitcoinsv.io/index.php/OP_RETURN) opcode that allows adding extra information to transactions beyond standard inputs and outputs. This can be useful for applications that want to include:
+
+* Different transaction types related to events (e.g., gaming);
+
+* Invoice numbers;
+
+* Nonces to differentiate payments of the same amount;
+
+* Other use cases.
```

Comment:
"Other data". Otherwise the sentence becomes "This can be useful for applications that want to include **other use cases**" which doesn't make much sense.